packages/commonwealth/scripts/cosmos_testnet_startup.sh

```diff
@@ -0,0 +1,54 @@
+# Following steps from: https://github.com/cosmos/cosmos-sdk/tree/v0.46.11/simapp
+
+CSDK_HOME=$HOME/.simapp
+CONFIG_FOLDER=$CSDK_HOME/config
+GENESIS=$CONFIG_FOLDER/genesis.json
+# You can add this mnemonic to keplr to use the UI
+COW_MNEMONIC="ignore medal pitch lesson catch stadium victory jewel first stairs humble excuse scrap clutch cup daughter bench length sell goose deliver critic favorite thought"
+
+#  https://daseldocs.tomwright.me/ used to edit genesis.json
+brew install dasel
```

Comment:
Can we get this working for linux as well? I'm sadly not a Mac user.

Reply:
Should not be a blocker to shipping, but absolutely should be fixed going forward (I would also like to be doing more development & testing on my Linux machine, rather than my Macbook.) Let's discuss in office hours Thursday.

Reply:
this is now set up for the docker image, so it will be linux friendly. But just need to `yarn cosmos:testnet` now to run docker compose up

Reply:
now `yarn cosmos:run` from chain-events/cosmos-chain-testing